pkgs/development/python-modules/typedunits/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  setuptools,
+  wheel,
+  attrs,
+  numpy,
+  protobuf,
+  pyparsing,
+}:
+
+buildPythonPackage {
+  pname = "typedunits";
+  version = "unstable-2025-05-09";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format

There are no upstream versions yet. Would it make more sense to just build this as a subpackage of `python3Packages.cirq-google`?

Reply:
What am I not seeing about the version number?

Reply:
> If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning or tagging scheme - then the latest upstream version in the above schema should be 0.

Reply:
Ah. Thank you.